map/src/util/dateFmt.js

```diff
@@ -1,6 +1,7 @@
 import i18n from 'i18next';
+import { normalizeLang } from '../i18n';
 
-const lang = () => i18n?.language || (typeof navigator !== 'undefined' ? navigator.language : 'en-US');
+const lang = () => normalizeLang(i18n?.language) || (typeof navigator !== 'undefined' ? navigator.language : 'en-US');
```

Comment:
This approach is wrong. i18n?.language is the current language and should already be normalized, so calling normalizeLang(i18n?.language) is incorrect